corehq/toggles.py

```diff
@@ -375,3 +375,9 @@ def toggles_dict(username=None, domain=None):
      "because it has to be accompanied by a migration.)"),
     [NAMESPACE_DOMAIN],
 )
+
+LOGISTICS_CUSTOM_CONSUMPTION = StaticToggle(
+    'LOGISTICS_CUSTOM_CONSUMPTION',
+    'Enable custom way of compute consumption that ignores invalid soh reports',
+    [NAMESPACE_DOMAIN]
+)
```

Comment:
as per above I think it could make more sense to add to the domain consumption settings: https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/commtrack/models.py#L137